mypy_whitelist.txt

```diff
@@ -0,0 +1,4 @@
+pandas/core/arrays/base.py
```

Comment:
can you add some comments here to indicate what this file is. Also can you add a small new doc section on typing (just a starter is good, can expand over time). on how to type / how to update this file etc.